projects/components/src/datagrid/datagrid.component.html

```diff
@@ -49,7 +49,7 @@ <h3 *ngIf="this.header" class="vcd-header">{{ header }}</h3>
     </clr-dg-column>
 
     <clr-dg-row
-        *ngFor="let restItem of items; let i = index"
+        *ngFor="let restItem of items; let i = index; trackBy: trackBy"
```

Comment:
Can we add a comment here? It's unclear why we need both trackBy and `ngForTrackBy`